docs/UserGuide.md

```diff
@@ -157,57 +169,58 @@ Examples:
 * `add n/Bobby Tay p/9678 5432 e/BobbyTay@gmail.com a/Blk 123 Foo Street 45, #07-89 t/PRUactiveCash`
 * `add n/Betsy Crowe t/PRUactiveCash e/betsycrowe@example.com a/Block 82 Marine Parade Central #01-600 p/9876 5432 t/PRUTravellerProtect`
 
-Specifications:
-* `p/PHONE_NUMBER` has to be unique from the other contacts in FINDvisor.
-
-### Listing all persons : `list`
+### Listing all persons: `list`
 
 Shows a list of all persons in the contact list of FINDvisor.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing a person: `edit`
 
-Edits an existing person's contact information (name, phone number, email and address) and tags in FINDvisor.
+Edits an existing person's contact information (name, phone number, email, and address) and tags in FINDvisor.
 
 Format: `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 
 * Edits the person at the specified `INDEX`.
-* The index refers to the index number shown in the current displayed person list.
+* The index refers to the index number shown in the currently displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Input values will overwrite **all** existing values that were assigned to the specified field.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
+* When editing tags, **existing tags will be overwritten** with the new tags given.
 * You can remove all the person’s tags by typing `t/` without specifying any tags after it.
-* This command will check for duplicate mobile phone number before edit is executed. If the new mobile phone number is used by another person, the command will not be executed.
+* This command will check for duplicate mobile phone numbers before the `edit` command is executed. If the new mobile phone number is used by another person, the command will not be executed.
+
+<div style="page-break-after: always;"></div>
 
 Examples:
-* `edit 3 n/Bobby Tay e/bobbytay@u.nus.edu` Edits contact displayed at index 3 and changes the name of the contact to `Bobby Tay` and email to `bobbytay@u.nus.edu` respectively.
-* `edit 1 t/PRUActiveSaverIII t/PRUActiveCash` Edits contact displayed at index 1 to change its tags to only `PRUActiveSaverIII` and `PRUActiveCash`. It will remove all other previous tags that are previously associated with the contact.
+* `edit 3 n/Bobby Tay e/bobbytay@u.nus.edu` edits contact displayed at index 3 and changes the name of the contact to `Bobby Tay` and email to `bobbytay@u.nus.edu` respectively.
+* `edit 1 t/PRUActiveSaverIII t/PRUActiveCash` edits contact displayed at index 1 to change its tags to only `PRUActiveSaverIII` and `PRUActiveCash`. It will remove all other previous tags that are previously associated with the contact.
```

Comment:
Would  it be better to bold the "only" here to serve as a kind of warning. Also, I think the second sentence can be rephrased. The changes I propose are as follows:

```suggestion
* `edit 1 t/PRUActiveSaverIII t/PRUActiveCash` edits contact displayed at index 1 to change its tags to **only** `PRUActiveSaverIII` and `PRUActiveCash`. It will remove **all** other tags that were previously associated with the contact.
```